dimod/reference/composites/cutvertex.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+"""
+A composite that breaks the problem into sub-problems corresponding to the
+biconnected components of the binary quadratic model graph before sending to its child sampler.
+"""
+
+from dimod.sampleset import as_samples
+from dimod.core.composite import ComposedSampler
+from dimod.sampleset import SampleSet
+import dimod
+import networkx as nx
+import itertools
+
+
+__all__ = ['CutVertexComposite']
+
+
+class CutVertexComposite(ComposedSampler):
```

Comment:
To make things easy to search for, would it be too weird to have an alias like `CutVertexComposite = CutVariableComposite`. Just because "cut vertex" is a more likely search term.